storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -321,6 +321,12 @@
      */
     @IsBoolean
     public static final String TOPOLOGY_RAS_ONE_EXECUTOR_PER_WORKER = "topology.ras.one.executor.per.worker";
+    /**
+     * Whether to limit each worker to one component. This is useful for debugging topologies to clearly identify workers that
+     * are slow/crashing and for estimating resource requirements and capacity.
+     */
```

Comment:
Also I think we can add some comment here to be more clear about which one will be used if both of these are set true.